scripts/send_delta_emails.gmp

```diff
@@ -81,46 +82,46 @@ print()
 
 
 def execute_send_delta_emails(sc):
-    print ('Retrieving task list ...')
+    print('Retrieving task list ...')
     task_filter = 'tag=%s' % task_tag
     tasks = gmp.get_tasks(filter=task_filter).xpath('task')
-    print ('Found %d task(s) with tag "%s".' % (len(tasks), task_tag))
+    print('Found %d task(s) with tag "%s".' % (len(tasks), task_tag))
 
     for task in tasks:
         task_id = task.xpath('@id')[0]
-        print ('Processing task "%s" (%s)...' % (task.xpath('name/text()')[0], task_id))
+        print('Processing task "%s" (%s)...' % (task.xpath('name/text()')[0], task_id))
 
         reports = gmp.get_reports(report_filter='task_id=%s and        \
                                                 status=Done            \
                                                 sort-reverse=date'
-                                                % task_id).xpath('report')
-        print ('  Found %d report(s).' % len(reports))
-        if (len(reports) < 2):
-            print ('  Delta-reporting requires at least 2 finished reports.')
+                                  % task_id).xpath('report')
+        print('  Found %d report(s).' % len(reports))
+        if len(reports) < 2:
+            print('  Delta-reporting requires at least 2 finished reports.')
             continue
 
         if reports[0].xpath('report/user_tags/tag/name[text()="delta_alert_sent"]'):
-            print ('  Delta report for latest finished report already sent')
+            print('  Delta report for latest finished report already sent')
             continue
 
-        print ("  Latest finished report not send yet. Preparing delta report...")
+        print("  Latest finished report not send yet. Preparing delta report...")
 
         delta_report = gmp.get_reports(
-                                       report_id=reports[0].xpath('@id')[0],
-                                       delta_report_id=reports[1].xpath('@id')[0],
-                                       filter='delta_states=n',
-                                       format_id='c1645568-627a-11e3-a660-406186ea4fc5'
-                                       )
+            report_id=reports[0].xpath('@id')[0],
+            delta_report_id=reports[1].xpath('@id')[0],
+            filter='delta_states=n',
+            format_id='c1645568-627a-11e3-a660-406186ea4fc5'
+            )
 
         csv_in_b64 = delta_report.xpath('report/text()')[0]
         csv = base64.b64decode(csv_in_b64)
         # Only for debugging:
-        #print(csv)
+        # print(csv)

```

Comment:
i'm unsure about this. it is not a normal comment, so it feels right as it was. such lines probably shouldn't be in production code at all. so maybe remove it completely if not needed anymore or "make it official" with something like this, already used in other files:

`debug = True  # flag for debugging information output`

i can also do research what the best practice aka pythonic way is...

Reply:
Agreed. My change was directed more at the style, with less attention to the content.

> such lines probably shouldn't be in production code at all. so maybe remove it completely if not needed

I think this would be the best solution. GMP scripts should be expected to not require code edits in operation. If output of debug information is desired, I would prefer if it could be controlled consistently via `gvm_pyshell`, e.g. by using the `logging` library and honoring the corresponding setting passed to `gvm_pyshell`.

Reply:
You can propose deleting the 2 cases where debug code is still present in an additional PR and let Joshua (who introduced them) review it.

Reply:
Ack, sounds like a plan.